code/sorting/src/bucket_sort/README.md

```diff
@@ -1,4 +1,36 @@
-# cosmos
-Your personal library of every algorithm and data structure code that you will ever encounter.
+# Bucket Sort
+Bucket sort is a sorting algorithm, that works by distributing the elements of an array into a number of buckets. Then, we apply some sorting algorithm (**insertion sort** for best optimisation ) to sort elements in each bucket, & finally, take the elements out of buckets in order & put them back to original array to get sorted result.
```

Comment:
1. No need for the first comma.
2. Then, we apply _a_ sorting algorithm
3. **insertion sort** for best performance (Or optimization if you want to keep the word)
4. Better to use and than &
5. get _the_ sorted result.